backends/dpdk/dpdkArch.h

```diff
@@ -675,6 +675,13 @@ class CollectExternDeclaration : public Inspector {
                     ::error(ErrorType::ERR_EXPECTED,
                             "%1%: expected size and optionally init_val as arguments", d);
                 }
+            } else if (externTypeName == "MatchValueLookupTable") {
+                if (d->arguments->size() != 1 &&
```

Comment:
Hope you check can for boundaries ( > 0 && <= 3) instead of three individual comparisons.